publishRelease.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+RELEASE_VERSION="$(./gradlew -q -PpublishReleaseVersion printVersion)"
```

Comment:
this won't work because we don't have gradle here. Try this instead

`RELEASE_VERSION="$(grep -F "<version>" pom.xml | head -n 1 | sed 's/<[^>]*>//g' | sed 's/ //g')"`